package.json

```diff
@@ -62,6 +62,7 @@
     "react-router": "~2.8.1",
     "react-router-redux": "~4.0.6",
     "react-tap-event-plugin": "~2.0.0",
+~    "recompose": "^0.21.2",
```

Comment:
sorry, the sed command interpreted the `^`... I meant to replace it with a tilde